packages/spec/integration/typescript/package.json

```diff
@@ -18,5 +18,8 @@
   "dependencies": {
     "hops": "*",
     "hops-typescript": "*"
+  },
+  "devDependencies": {
```

Comment:
It's not a big deal but I think it would be "more correct" do treat `typescript` as a normal dependency.